maintainer/conda/environment.yml

```diff
@@ -0,0 +1,36 @@
+name: mda-dev
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - chemfiles
+  - codecov
+  - cython
+  - griddataformats
+  - gsd
+  - h5py==2.10.0
+  - hypothesis
+  - joblib>=0.12
+  - matplotlib==3.2.2
+  - mmtf-python
+  - mock
+  - networkx
+  - numpy>=1.17.3
+  - psutil
+  - pytest
+  - scikit-learn
+  - scipy
+  - pip
+  - plotly
```

Comment:
This is not a full list of dependencies (and actually, `plotly` shouldn't be here). When all dependencies were given, RTD errored out with a memory and time error on installation. Even now it's quite slow; if it happens more in the future we could probably remove everything but cython, numpy, and the sphinx/msmb_theme packages.